SAP/Script/FXVProduct.py

```diff
@@ -105,7 +104,7 @@ def sap_test():
       
     #Price  
     price = frame.FindElement("#configurationComponent---configurationView--headerContainer-2--headerFieldPrice").text
-    record_price(price, record_no_values(sap_field_values), general_requirement_values(sap_field_values))
+    record_price(price,sales_order_values(sap_field_values),general_requirement_values(sap_field_values))
```

Comment:
user direct Count value

Reply:
removed the pervious parameter and pass the direct count's Json value